### PR TITLE
Refactor main menu on top bar

### PR DESCRIPTION
**Pull request by skrzepij**
Move from antd Navbar to custom one

### CODE REVIEW

File: src/components/MainHeaderComponent/MainHeaderComponent.tsx [hunk 1]
```diff
@@ -27,44 +26,22 @@ const MainHeader = () => {
...
+            <ul className="main-header__menu">
+              <HeaderNavLink
+                icon={<BuildFilled />}
+                route={menuRoutes.about()}
+              />
+              <HeaderNavLink
+                icon={<ContainerFilled />}
+                route={menuRoutes.contact()}
+              />
+            </ul>
```
Comment (skrzepij) [inline src/components/MainHeaderComponent/MainHeaderComponent.tsx:43]: should we extract this `menu` to new component? 🤔

File: src/components/MainHeaderComponent/MainHeaderComponent.scss [hunk 2]
```diff
@@ -21,66 +20,83 @@
...
+      width: 100%;
+      text-align: center;
+      max-height: 0;
+      line-height: 40px;
+      transition: max-height .3s ease-in-out;
+      &::before, &::after {
+        content: '';
+        display: block;
+        margin: 0 -20px;
+        background-color: $cc-white;
```
Comment (skrzepij) [inline src/components/MainHeaderComponent/MainHeaderComponent.scss:57]: First background-color, next margin.

File: src/components/MainHeaderComponent/MainHeaderComponent.scss [hunk 3]
```diff
@@ -21,66 +20,83 @@
...
+        font-size: .6rem;
       }
+    }
 
-      .ant-menu {
-        padding: 0;
+    &__menu {
+      order: 3;
+      width: 100%;
+      text-align: center;
```
Comment (skrzepij) [inline src/components/MainHeaderComponent/MainHeaderComponent.scss]: `text-align` should not be between `width` and `height` settings

File: src/components/HeaderNavLink/HeaderNavLink.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+export const HeaderNavLink: React.FC<Props> = ({icon, route}) => {
+  const {t} = useTranslation();
+
+  return (
+    <li className={styles.navLink}>
+      <NavLink activeClassName={styles.active} to={route.path}>
+        {icon}
+        <span className={styles.navLink__label} data-hover={t(route.label)}>
+                      {t(route.label)}
+                    </span>
```
Comment (skrzepij) [inline src/components/HeaderNavLink/HeaderNavLink.tsx]: emm isn't there something wrong with the formatting? Oo

File: src/components/ChangeLanguage/ChangeLanguage.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+const {Option} = Select
+
+export const ChangeLanguage: React.FC = () => {
+  const {i18n} = useTranslation();
+
+  const onLanguageChange = (lang: AvailableLanguageCodes) => i18n.changeLanguage(lang)
+
+  return (
+    <div className={styles.changeLanguage}>
+      <label htmlFor="languageSelect" className="sr-only">Select language</label>
```
Comment (skrzepij) [inline src/components/ChangeLanguage/ChangeLanguage.tsx]: this is for screen readers only, but it should be translated after all.

File: src/components/MainHeaderComponent/MainHeaderComponent.tsx (hunk 1 above)
Comment (skrzepij) [reply src/components/MainHeaderComponent/MainHeaderComponent.tsx:43]: naaah no need to right now.

**Review by MateuszWasik — APPROVED** [review]

**Pull request merged**